### DISCUSSION TITLE
Flight Heritage of Fprime

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/fprime#1376] [Q&A]
Hello

What is the flight heritage of FPrime SW ? Is it used in any of the commercial missions and any such samples available ?

**Comment by Emil808** [comment]
Section 3 of [this document](https://s3vi.ndc.nasa.gov/ssri-kb/static/resources/CL18-2993.pdf). The document presents the Fprime framework and describes a couple projects that JPL has done with it.
